ui/src/containers/AppDetailsToolbarContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { AppDetailToolbar } from '../components/AppDetailToolbar';
+
+export class AppDetailsToolbarContainer extends React.Component {
+  onDisableButton = () => {
```

Comment:
Could be named `onDisableApp` to be more descriptive. It is an event handler which could technically be used by something other that a button.